identity-wallet/src/state/qr_code/reducers/read_credential_offer.rs

```diff
@@ -144,7 +144,7 @@ pub async fn read_credential_offer(state: AppState, action: Action) -> Result<Ap
 async fn download_credential_logos(
     credential_configurations: &HashMap<String, CredentialConfigurationsSupportedObject>,
 ) {
-    for (credential_configuration_id, credential_configuration) in credential_configurations.iter() {
+    for (_, credential_configuration) in credential_configurations.iter() {
```

Comment:
Minor thing, but since we don't use the keys here anymore I guess this would be slightly cleaner:
```rust
for credential_configuration in credential_configurations.values()
```